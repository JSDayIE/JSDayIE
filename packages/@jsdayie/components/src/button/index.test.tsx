import React from 'react';
import renderer from 'react-test-renderer';
import { Button } from '.';

test("Button", async () => {
    const component = renderer.create(
        <Button href="/somepath" color="warning">Test</Button>
      );
      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
});