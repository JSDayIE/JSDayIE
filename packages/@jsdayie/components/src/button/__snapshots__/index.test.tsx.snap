// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button 1`] = `
<a
  className="btn btn-outline-warning btn-lg "
  href="/somepath"
  rel="noreferrer"
  target="_blank"
>
  Test
</a>
`;
